modules/nodes-safe-sysctls-list.adoc

```diff
@@ -0,0 +1,123 @@
+
+All unsafe sysctls are not allowed by default. For system-wide sysctls the cluster administrator must manually enable them on a per-node basis. Pods with disabled unsafe sysctls are scheduled but do not launch.
+
+[NOTE]
+====
+You cannot manually enable interface specific unsafe sysctls.
+====
+
+{product-title} adds the following system-wide and interface specific safe sysctls to an allowed safe list:
+
+.System-wide safe sysctls
+[cols="30%,70%",options="header"]
+|===
+| sysctl | Description
+
+| `kernel.shm_rmid_forced`
+a|When set to 1, all
+shared memory objects in current ipc namespace will be automatically forced to use IPC_RMID.
+For more information see link:https://docs.kernel.org/admin-guide/sysctl/kernel.html?highlight=shm_rmid_forced#shm-rmid-forced[shm_rmid_forced].
+
+| `net.ipv4.ip_local_port_range`
+a| Defines the local port range that is used by TCP and UDP to choose the local port. The first number is the first, the second the last local port number. If possible, it is better these numbers have different parity (one even and one odd value). Must be greater than or equal to ip_unprivileged_port_start. The default values are 32768 and 60999 respectively.
+For more information see link:https://docs.kernel.org/networking/ip-sysctl.html?highlight=ip_local_port_range#ip-variables[ip_local_port_range].
+
+| `net.ipv4.tcp_syncookies`
+|When net.ipv4.tcp_syncookies is set, the kernel will handle TCP SYN packets normally until the
+half-open connection queue is full, at which time, the SYN cookie functionality kicks in. This functionality SYN cookies allows the system to keep accepting valid connections, even if
+under a denial of service attack. For more information see link:https://docs.kernel.org/networking/ip-sysctl.html?highlight=tcp_syncookies#tcp-variables[tcp_syncookies].
+
+| `net.ipv4.ping_group_range`
+a| This restricts ICMP_PROTO datagram sockets to users in the group range. The default is “1 0”, meaning, that nobody (not even root) may create ping sockets. For more information see link:https://docs.kernel.org/networking/ip-sysctl.html?highlight=ping_group_range#ip-variables[ping_group_range].
+
+| `net.ipv4.ip_unprivileged_port_start`
+| This defines the first unprivileged port in the network namespace. To disable all privileged ports, set this to 0. They must not overlap with the ip_local_port_range. For more information see link:https://docs.kernel.org/networking/ip-sysctl.html?highlight=ip_unprivileged_port_start#ip-variables#ip-variables[ip_unprivileged_port_start].
+|===
+
+
+.Interface specific safe sysctls
+[cols="30%,70%",options="header"]
+|===
+| sysctl | Description
+
+| `net.ipv4.conf.IFNAME.accept_ra`
+a|Accept IPv4 Router Advertisements; autoconfigure using them. It also determines whether or not to transmit router solicitations. If and only if the functional setting is to accept Router Advertisements, Router Solicitations will be transmitted.
```

Comment:
```suggestion
a|Accept IPv4 Router Advertisements; autoconfigure using them. It also determines whether or not to transmit router solicitations. Router Solicitations will be transmitted only if the functional setting is to accept Router Advertisements.
```

Reply:
- Agree but "Router Solicitations will be transmitted" -> "Router Solicitations are transmitted"
- Are "Router Advertisements" and "Router Solicitations" proper nouns? Not sure they should be capitalized.